### DISCUSSION TITLE
Keycloak for authentication/authorization on an external database

### DISCUSSION THREAD

**Discussion by TestSubject1498** [keycloak/keycloak#14223] [Q&A]
I'm looking to use keycloak as a means of authentication and authorization for external database servers (IE the ["application/service"](https://www.keycloak.org/docs/latest/securing_apps/index.html#_oidc) I want to secure is a postgres database). Is there existing documentation on this?
I'm having difficulty finding any information securing access to external databases as my searches in the issues / documentation instead pull up results for configuring the keycloak database to persist realm and credentials information, etc. [(Such as this).](https://www.keycloak.org/server/db) 
If someone could point me in the right direction that would be very much appreciated.

**Comment by ayoubAnbara** [comment]
You want to link Keycloak with external database instead of using H2, or what?
